sklearn/utils/_array_api.py

```diff
@@ -232,6 +232,14 @@ def take(self, X, indices, *, axis=0):
     def isdtype(self, dtype, kind):
         return isdtype(dtype, kind, xp=self._namespace)
 
+    def bincount(self, X, weights=None, minlength=None):
+        count = numpy.bincount(X, weights=weights, minlength=minlength)
+        return self._namespace.asarray(count)
+
+    def searchsorted(self, a, v, *, side="left", sorter=None):
+        indices = numpy.searchsorted(a, v, side=side, sorter=sorter)
+        return self._namespace.asarray(indices)
+
```

Comment:
For these two methods since they only seem to be missing for the numpy.array_api and probably cupy.array_api is this format fine? We might have to convert to a cupy array maybe for cupy.array_api. For the time being I could not test out with CuPY because it does not seem to be available on Mac M1.